### PR TITLE
Add Object and Context Initializers and Mutators.

### PR DESCRIPTION
**Pull request by TravisWhitaker**
This commit adds SQL statements for initializing and mutating objects and
contexts. The code is by no means complete; I'm merely shoving something
out to provide an example of the patterns. Object and context accessors
come next. Naturally these will evolve as we go on.

### CODE REVIEW

File: src/CSH/Eval/DB/Statements.hs [hunk 1]
```diff
@@ -1 +1,394 @@
 module CSH.Eval.DB.Statements where
+
+import Database.HDBC
+
+import Database.HDBC.PostgreSQL
+
+data StatementGroup = StatementGroup {
+    asfdS :: Statement
```
Comment (gambogi) [inline src/CSH/Eval/DB/Statements.hs]: Will not merge until `asfdS` has a meaningful name.
Also, you need a comment explaining the why of this data structure

Comment (TravisWhitaker) [inline src/CSH/Eval/DB/Statements.hs]: Fixed.

**Comment by sdemos** [comment]
Looks good to me, but @gambogi should check it out too, since it is a rather large commit.

**Comment by gambogi** [comment]
Also looks good to me; merging

**Pull request merged**